lookout/style/typos/benchmarks/evaluate_typos.py

```diff
@@ -47,4 +48,5 @@ def evaluate_typos_on_identifiers(dataset: str = TYPOS_DATASET,
                            Candidate=Candidate, Columns=Columns,
                            tokenize=lambda x: list(analyzer.parser.split(x)),
                            flatten_df_by_column=flatten_df_by_column,
-                           generate_report=generate_report)
+                           generate_report=generate_report,
+                           logical_and=numpy.logical_and, logical_not=numpy.logical_not)
```

Comment:
is it possible to avoid `logical_and` and `logical_not` passing? In numpy, you can use `&` as `logical_and` and `~` as `logical_not`. Do they work in Jinja2 template?

Reply:
Yep, looks like it works